src/project/FileViewController.js

```diff
@@ -160,8 +160,9 @@ define(function (require, exports, module) {
         _curDocChangedDueToMe = true;
 
         _fileSelectionFocus = fileSelectionFocus;
+        
 
-        paneId = (paneId || MainViewManager.ACTIVE_PANE);
+        paneId = (paneId || ((event.ctrlKey || event.metaKey) && event.altKey ? MainViewManager.SECOND_PANE : null) || ((event.ctrlKey || event.metaKey) ? MainViewManager.FIRST_PANE : null) || MainViewManager.ACTIVE_PANE);
```

Comment:
Could (should?) this check be it's own method, for example `FileViewController#_getPaneId()`?